tests/test_thread.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
+import os.path
+import subprocess
+import tempfile
+import unittest
+
+from drgn import Program
+from tests import TestCase
+
+
+class TestCoreDump(TestCase):
+    TIDS = (
+        2265413,
+        2265414,
+        2265415,
+        2265416,
+        2265417,
+        2265418,
+        2265419,
+        2265420,
+        2265421,
+        2265422,
+        2265423,
+        2265424,
+        2265425,
+    )
+
+    CRASHED_TID = 2265419
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        with tempfile.NamedTemporaryFile() as core_dump_file:
+            try:
+                subprocess.check_call(
+                    [
+                        "zstd",
+                        "--quiet",
+                        "--decompress",
+                        "--stdout",
+                        "tests/sample.coredump.zst",
+                    ],
+                    stdout=core_dump_file,
+                )
+            except FileNotFoundError:
+                raise unittest.SkipTest("zstd not found")
+            cls.prog = Program()
+            cls.prog.set_core_dump(core_dump_file.name)
+
+    def test_threads(self):
+        self.assertEqual(
```

Comment:
You could use `assertSequenceEqual` instead, and then you don't have to convert `self.TIDS` to a `list`.

```suggestion
        self.assertSequenceEqual(
```